Rearrange pixel corner co-ordinates

Adimote opened the pull request:
Rearrange the marker corners to match that of apriltags

Adimote committed:
Rearrange corner locations to match Apriltags

The Previous API had marker corners ordered Clockwise from the bottom left.

Apriltags orders them clockwise from the top left.

This change converts the API to match Apriltags' output to reduce
code complexity.

Adimote committed:
add comment to pixel_corners

PeterJCLaw commented:
@Adimote Are these relative to the marker or the image?

That is, if the marker is upside down: is the first position value here that of the upper-leftmost corner within the image or the bottom-rightmost corner in the image?

Adimote commented:
@PeterJCLaw yes that's correct

Adimote committed:
mention the top left is relative to the marker

Adimote commented:
@PeterJCLaw I've made it more clear in the comment

PeterJCLaw reviewed (approved):
Even better would be to include an example explaining what it means for the coordinates to be relative to the marker.

Adimote committed:
Fix pixel corner co-ordinates ordering

Turns out the pixel corner co-ordinates ordering was incorrect this fixes that.

Adimote committed:
Add example for the first pixel corner value

Adimote commented:
Turns out I was completely incorrect about the ordering of the corners. That shows me for trusting my memory. I've now updated the ordering to match actual values

PeterJCLaw reviewed (approved):
Given the level of uncertainty around this, maybe a test to demonstrate example values is in order?

PeterJCLaw commented:
Speaking of testing -- the CI is highlighting some long lines.
